docs/support/faq.md

```diff
@@ -6,6 +6,19 @@ order: 60
 
 # Frequently asked questions
 
+## How to set the height of CKEditor 5?
+
+The height of the editing area can be easily controlled with CSS. For example, the `minHeight` and `maxHeight` settings can be set with the following code:
```

Comment:
```suggestion
The height of the editing area can be easily controlled with CSS.
```

I would remove the sentence about `minHeight` and `maxHeight` as these properties are not valid CSS or CKEditor 5 properties.